packages/hubble-sdk/tests/sdk.test.ts

```diff
@@ -237,4 +237,24 @@ describe('Hubble SDK Tests', () => {
   //   console.log(circulatingSupply);
   //   expect(circulatingSupply.isZero()).toBeFalsy();
   // });
+
+  //   test('should get user loans', async () => {
```

Comment:
why did we comment this?

Reply:
We, unfortunately, don't have integration tests set up in the CI/CD here so I just test it out locally. No time to set it up and since we already test the actual functionality in the smart contracts repository, we did not find it worth it to set everything up.